src/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect, useRef } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import Icon from "../Icon/Icon";
+import Check from "../Icon/Icons/components/Check";
+import "./Checkbox.scss";
+
+const BASE_CLASS_NAME = "monday-style-checkbox";
+
+export const Checkbox = ({
+  componentClassName,
+  label,
+  onChange,
+  checked,
+  disabled,
+  defaultChecked,
+  value,
+  name
+}) => {
+  const checkboxClassNames = [`${BASE_CLASS_NAME}__checkbox`];
+  let overrideDefaultChecked = defaultChecked;
+
+  // If component did not receive default checked and checked props, choose default checked as
+  // default behavior (handle isChecked logic inside input) and set default value
+  if (overrideDefaultChecked == undefined && checked == undefined) {
```

Comment:
why "==" ? and not "==="

Reply:
Because it check both undefined and null

Reply:
we prefer not to use the Abstract Equality Operator ("==") I think we can change it to be more explicit or use lodash isNil in my opinion it is clearer